johnwayodi/sm-server
# [Feature #161873520] Add signout capability, single admin login, fix tests

#### What does this PR do?
Add Signout Capability for both the administrator and the attendant
Add a check so that only one administrator is able to register
#### Description of Task to be completed?
1. Add logout endpoint to the api
2. Add check to prevent more than one person from registering as the administrator.
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api-v2.git`
2. Ensure you are in the cloned repo, **sm-api-v2** by default.
3. Create a virtual environment using virtualenv and `pip install -r requirements.txt`
4. `git checkout -b ft-user-logout-161873520 origin/ft-user-logout-161873520`
5. Run **_pytest -vv --cov=storemanager/api/v2_** in terminal to run the tests and get coverage on the application.
#### Any background context you want to provide?
No background context
#### What are the relevant pivotal tracker stories?
[#161873520](https://www.pivotaltracker.com/story/show/161873520)
#### Questions:
None for this section
